ref.yml

```diff
@@ -392,29 +399,191 @@ fields:
+        The tasks must be described in either:
+        - the current `meta.yml` through the attribute: `nix-tasks` (see below),
+          the result of which is located in `./nix/fallback-config.nix`.
+        - or a user-created `.nix/config.nix` (which takes priority over
+          `./nix/fallback-config.nix` and the `meta.yml` description of
+          `nix-tasks`, so use with caution).
+        Otherwise, this attribute cannot be of any use.
+        /!\ It is strongly recommended to activate cachix with at least
+        /!\ the coq cachix activated `cachix: [ {"name": "coq" } ]``
+      type: list
+      item_fields:
+        - job:
+            required: false
+            type: string
+            description: |
+              The name of the job as described in `.nix/config.nix`
+              as in e.g. tasks.<t>.coqPackages.<mypkg>.ci.job = <job>;
+              The default is a `all` job which will run **everything**
+            default: "all"
+        - name:
+            required: false
+            type: string
+            description: |
+              The name to display in the github action web interface.
+              Defaults to the job above.
+            example: "My job"
+        - needs:
+            required: false
+            type: list
+            description: |
+              The list of jobs to wait for before executing these ones.
+            default: "main"
+            item_fields:
+              required: false
+              type: string
+              description: A job to wait for
+  - nix-default-task:
+      used:
+        - fallback-config.nix
+      required: false
+      description: Default task to use when typing nix-shell without arguments
+      type: string
+      default: "default"
+  - nix-tasks:
+      used:
+        - fallback-config.nix
+      description: |
+        Describe a set of tasks to be performed in parallel for each job.
+        Suitable to test several versions of dependencies and reverse
+        dependencies.
+      type: list
+      item_fields:
+        - task:
+            required: true
+            type: string
+            description: The name of the task
+        - packages:
+            required: false
+            type: list
+            description: Package to override or add to CI
+            item_fields:
+              - name:
+                  required: true
+                  description: Name of the package attribute from nixpkgs
+                  type: string
+              - from:
+                  required: false
+                  type: string
+                  description: Where to find the attribute
+                  default: coqPackages
+                  examples:
+                    - "coqPackages"
+                    - "ocamlPackages"
+              - version:
+                  required: false
+                  type: string
+                  description: Version number (release, PR, branch or URL)
```

Comment:
Might be worth adding a reference to https://nixos.org/manual/nixpkgs/unstable/#sec-language-coq

Reply:
```suggestion
                  description: Version number (release, PR, or branch), as documented in
                               https://nixos.org/manual/nixpkgs/unstable/#sec-language-coq
```